ga4gh/datamodel/datasets.py

```diff
@@ -180,6 +226,26 @@ def __init__(self, localId, dataDir, backend):
                     self, localId, bamPath, backend)
                 self.addReadGroupSet(readGroupSet)
 
+        # Rna Quantification
+        rnaQuantDir = os.path.join(dataDir, "rnaQuant")
+        for rnaQuantId in os.listdir(rnaQuantDir):
+            relativePath = os.path.join(rnaQuantDir, rnaQuantId)
+            if os.path.isdir(relativePath):
+                rnaQuantification = rna_quantification.RNASeqResult(
+                    self, rnaQuantId, relativePath)
+                self._rnaQuantificationIdMap[rnaQuantId] = rnaQuantification
```

Comment:
it would be good to have a `addRnaQuantification` method, a la the methods that are provided for other objects (see above in the file)... same goes for other objects you've added